test/extended/fixtures/test-sti-no-outputname.json

```diff
@@ -0,0 +1,35 @@
+{
+  "kind":"BuildConfig",
+  "apiVersion":"v1",
+  "metadata":{
+    "name":"test",
+    "labels":{
+      "name":"test"
+    }
+  },
+  "spec":{
+    "triggers":[],
+    "source":{
+      "type":"Git",
+      "git":{
+        "uri":"https://github.com/openshift/origin"
+      },
+      "contextDir":"test/extended/fixtures/test-build-app"
+    },
+    "strategy":{
+      "type":"Source",
+      "env": [
+        {
+          "name": "BUILD_LOGLEVEL",
```

Comment:
@bparees we should make this default for tests.... Also I was thinking that `start-build` should have `--build-loglevel` parameter that will set this env up for you automatically. That will allow nicer testing:

```
$ oc start-build --build-loglevel=5 test
```